.github/workflows/perf.yml

```diff
@@ -0,0 +1,50 @@
+# runs some very-large-repo tests (for windows filesystem limits) and commits perf results for comparison
+name: perf-tests
+on:
+  push:
+    branches-ignore: [main, gh-pages]
+  workflow_dispatch:
+
+# linux will finish ahead of windows, but prevent other branches/commits from hitting simultaneously
+# since we're pushing git commits and there would be conflicts
+concurrency: perf-test
+
+jobs:
+  perf-tests:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: 'npm'
+
+      # Install dependencies
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run compile
+
+      - name: Run performance tests
+        run: npm run test:perf | tee test/perf/output.txt
```

Comment:
Do we need this workflow? I don't think we even have test:perf script

Reply:
Nope